### PR TITLE
List structure/feature

### PR DESCRIPTION
**Pull request by denscholar**
## Features Implemented

The following features were implemented as per requirement:

- A new project was set up with webpack .
- An index.html file was created.
-  An index.js file was created with an array of some simple to do tasks containing three keys:
1. description [string].
2. completed [bool].
3. index: [number].

- A function was written to iterate over the tasks array and populate an HTML list item element for each task.
- On page load, the data is rendered dynamically in the dedicated placeholder. 
- All source files (index.html, index.js and style.css) is located in /src directory

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    completed: false,
+    index: 6,
+  },
+];
+
+const displayTodo = () => {
+  todoTasks.forEach((todo) => {
+    const todoItem = document.createElement('li');
+    todoItem.innerHTML = `
+          <form class="form-container">
```
Comment (moise10r) [inline src/index.js:55]: - [ ] Great job iterating over the tasks array and populating an HTML list item element for each task :clap: however The list should appear in order of the index values as required, you can do it by sorting your `todoTasks `array before using forEach method

eg: 
``` 
export const sortItems = (items) => {
  const sortedItems = items.sort((a, b) => (a.index - b.index));
  return sortedItems;
};
```

**Review by moise10r — CHANGES_REQUESTED** [review]
### Hi **@denscholar,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- GitHubFlow was correctly used.  ✔️   
- Good commit messages ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (denscholar) [reply src/index.js:55]: @moise10r Thank you for your feedback. However, do I need to sort them again when already the index is already in an ascending (sorted) order? Unless I don't understand what you meant, in that case, you might want to further explain why I have to sort it again even when the index value is already sorted. Thank you.

**Review by Davidosky007 — COMMENTED** [review]
Hello @denscholar  👋,

_This is an additional review as a result of the second opinion request._

According to the project requirement, i can see that the items you are rendering from the array of tasks are in order of the index values for each task. 
image


As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @denscholar, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**